domain-cc/README.md

```diff
@@ -33,3 +36,13 @@ pip install -r dev-requirements.txt
 # MAKE SURE you are in adb-vro/domain-cc to get the right pre-commit-config.yaml installed
 pre-commit install
 ```
+
+## Building docs
+```
+source ~/.virtualenvs/domain-cc/bin/activate
+cd src/
+uvicorn api:app --reload &
```

Comment:
Nitpick -- for folks not used to shell backgrounding, maybe add an extra line or comment at the end for stopping `uvicorn` once docs are built?